src/server/v3/resources/remote_build_nodes.py

```diff
@@ -48,12 +49,15 @@ def get(self, remote_build_node_xname):
         log_id = get_log_id()
         current_app.logger.info("%s ++ remote_build_status.v3.GET %s", log_id, remote_build_node_xname)
 
+        # verify that the remote build node ssh key is present
+        if not test_private_key_file(current_app):
+            current_app.logger.info("SSH key not present for remote build nodes")
+
```

Comment:
You don't bail here. Should you?

Reply:
No, since it needs to return a status object per remote node for the http call.
This is just entered into the ims log to help determine root cause if someone wants to look.

Reply:
Okay.